Resources/Backup/The Complete Documentation About Genesis Rom Format (v1.1).txt

```diff
@@ -1,375 +0,0 @@
-// NOTE: Backed up copy from http://www.emulatronia.com/doctec/consolas/megadrive/genesis_rom.txt.
```

Comment:
Why is this file deleted? Can we revert this?

Reply:
Reverted it.